Bearing/BearingChapterBuilder.cpp

```diff
@@ -53,13 +54,1106 @@ Uint16 CBearingChapterBuilder::GetMaxLevel() const
 
 rptChapter* CBearingChapterBuilder::Build(const std::shared_ptr<const WBFL::Reporting::ReportSpecification>& pRptSpec,Uint16 level) const
 {
+
```

Comment:
This is a huge function. Consider breaking this up into smaller functions, with each function reporting a specific aspect of the report. Functions can be private class methods or file scope methods. File scope is better. 

With smaller functions Build shows the high level organization of the report.

Something like this...
ReportIntroduction();
ReportBearingDetails();
ReportLoads();

Reply:
I made an attempt at this. See commit. From what I understand, they are file scope methods.